packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -130,7 +130,7 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     const isSticky = isStickyTop || isStickyBottom;
 
     return (
-      <div ref={ this._resolveRef('_root') }>
+      <div ref={ this._resolveRef('root') }>
```

Comment:
Since this couldn't have possibly worked, let's keep it as _root and make the variable private.